### DISCUSSION TITLE
What is the correct way to set image front matter in both doc and blog?

### DISCUSSION THREAD

**Discussion by Ouch1978** [facebook/docusaurus#8155] [Q&A] [answered]
I had a trouble when I was setting the image property in front matter, and would like to know what is the best practice to do so.

Let say, I put my images along with my markdown file in the same folder, for example:

```
blog
  |-my-post
      |- index.md
      |- image-01.png
      |- image-02.png 
```

I could easily use the images in my markdown just with 
```markdown
![my-image-description](image-01.png)
```

Although the image will be moved to the static folder after build, the image still show up properly when I browse.

However, if I set the image in front matter with:
```markdown
image: image-01.png
```

It will be parsed to html meta tag:
```html
<meta data-rh="true" property="og:image" content="https://{my-base-url}/image-01.png">
```

The link is incorrect so that I could not get the preview image when I share my post to Facebook.

May I know what is the correct way if I would like to make og:image meta working properly without huge change?

Thank you.

**Comment by Josh-Cena** [comment, marked as answer]
You need a leading dot. `./image1.png`

**Reply by Ouch1978** [reply]
@Josh-Cena 
Thank you, it's working now.
